core/lib/generators/refinery/cms/cms_generator.rb

```diff
@@ -93,6 +94,10 @@ def append_heroku_gems!
 }.sub("{{production_gems}}", production_gems.join("\n  "))
     end
 
+    def current_ruby_version
+      (env_ruby_version = ENV['RUBY_VERSION']).present? ? env_ruby_version : RUBY_VERSION
```

Comment:
for this you can use:
```ruby
def current_ruby_version
  ENV['RUBY_VERSION'].presence || RUBY_VERSION
end
```

Reply:
Your right i under use `.presence`, my bad! it's fixed now